Snippets/Gateway/Gateway_3/Channels/Wildcard.config.xml

```diff
@@ -8,8 +8,10 @@
   <GatewayConfig>
     <Channels>
       <Channel Address="http://+:25899/RemoteSite/"
+               ChannelType="Http"/>               
+      <Channel Address="http://gateway.mycorp.com:25899/RemoteSite/"
```

Comment:
wouldn't that conflict with the strong wildcard which also goes to `xyz:25899/RemoteSite`?

Reply:
Good catch, will double check and adjust.

Tests seems to suggest that you are right

https://github.com/Particular/NServiceBus.Gateway/blob/develop/src/NServiceBus.Gateway.AcceptanceTests/When_listening_on_wildcard_uri.cs#L43

Reply:
Confirmed, different port is needed. Fixed now